### PR TITLE
Add instructions for implementing a concept exercise in elm

### PR DESCRIPTION
**Pull request by ceddlyburge**

### CODE REVIEW

File: languages/elm/docs/implementing-a-concept-exercise.md [hunk 1]
```diff
@@ -1,5 +1,119 @@
...
+      }
+    ]
+  }
+}
+```
+
+## Step 6: adding track-specific files
+
+Having added the files that are not specific to the track, now is the time to create the track-specific files. These file will include:
+
```
Comment (ErikSchierboom) [inline languages/elm/docs/implementing-a-concept-exercise.md]: It might be useful to list the actual file name when describing them below.

Comment (ceddlyburge) [reply languages/elm/docs/implementing-a-concept-exercise.md]: Hi Erik, I've added the filenames for these.

**Comment by ceddlyburge** [comment]
Hi Erik, I have a couple of questions.

1. Who merges this?
2. What do you want to do about the dead links?

The instructions, such as I can find them, state to create this file before creating issues for the concept exercises, and the concept exercises themselves, which I think leaves these options
1. Just accept the dead links
2. Link to example concepts from other exercies (for example, the csharp enum-advanced exercise)
3. Create a dummy / blank Elm concept exercise, and link to that.

File: languages/elm/docs/implementing-a-concept-exercise.md (hunk 1 above)
Comment (ErikSchierboom) [reply languages/elm/docs/implementing-a-concept-exercise.md]: LGTM!

**Comment by ErikSchierboom** [comment]
> Who merges this?

Once it is approved, you could merge. 

> What do you want to do about the dead links?

Perhaps just link to the C# ones for the moment, and keep in the todo to update it later once an Elm Concept Exercise has been added?

**Comment by ceddlyburge** [comment]
Hi @ErikSchierboom, I think the verify-links action isn't working. It seems to be getting stuck on a git conflict, but I have sorted out the conflict, and this pull request also shows that it can be merged.

I could just do it again, but it would be annoying and would need approving again and all that. And probably the link checker will need fixing anyway. Let me know if this is the best option though.

This is a good lesson in long lived branches / forks for me :)

**Comment by ErikSchierboom** [comment]
@ceddlyburge I've checked out this PR and tried to rebase on the latest master, and that does indeed give a merge conflict. Could you solve the merge conflict, which should fix the link checker.

**Comment by ceddlyburge** [comment]
Hi @ErikSchierboom, are you sure?

I have done this without any problems

```bash
git clone https://github.com/exercism/v3.git ev3
cd ev3
git checkout -b pull
git pull origin pull/779/head
git checkout master
git merge pull
```

The end result is this

```bash
Updating 55040eb..6791507
Fast-forward
 .../reference/implementing-a-concept-exercise.md   | 120 ++++++++++++++++++++-
 1 file changed, 118 insertions(+), 2 deletions(-)
```

Although at this point just starting again seems like it will probably be easier ...

**Comment by ErikSchierboom** [comment]
Weird! Shall I close this PR?

**Pull request closed without merging**